style/global.css

```diff
@@ -297,9 +297,12 @@ a {
   border-right: 1px solid #1f2325;
 }
 
-.tab>.icon {
-  width: 16px;
-  height: 16px;
+.tab>.file-icon {
+  vertical-align: middle;
+  text-align: center;
+  font-size: 90%;
+  min-width: 20px;
+  max-height: 20px;
```

Comment:
Tab icons require a bit of fixing to be aligned properly. Might be that not all rules are ultimately necessary, but that's where it started to look good.